website/docs/r/monitor_smart_detector_alert_rule.html.markdown

```diff
@@ -0,0 +1,109 @@
+---
+subcategory: "Monitor"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_monitor_smart_detector_alert_rule"
+description: |-
+  Manages an Monitor Smart Detector Alert Rule.
+---
+
+# azurerm_monitor_smart_detector_alert_rule
+
+Manages an Monitor Smart Detector Alert Rule.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_application_insights" "example" {
+  name                = "example-appinsights"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  application_type    = "web"
+}
+
+resource "azurerm_monitor_action_group" "example" {
+  name                = "example-action-group"
+  resource_group_name = azurerm_resource_group.example.name
+  short_name          = "exampleactiongroup"
+}
+
+resource "azurerm_monitor_smart_detector_alert_rule" "example" {
+  name                = "example-smart-detector-alert-rule"
+  resource_group_name = azurerm_resource_group.example.name
+  severity            = "Sev0"
+  scope               = [azurerm_application_insights.example.id]
+  frequency           = "PT1M"
+  detector_id         = "FailureAnomaliesDetector"
+
+  action_group {
+    ids = [azurerm_monitor_action_group.test.id]
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Monitor Smart Detector Alert Rule. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) Specifies the name of the resource group in which the Monitor Smart Detector Alert Rule should exist. Changing this forces a new resource to be created.
+
+* `detector_id` - (Required) Specifies the Built-In Smart Detector id that this alert rule will use. Currently possible values are `FailureAnomaliesDetector`.
+
+* `scope` - (Required) Specifies the scopes of this Smart Detector Alert Rule.
+
+* `action_group` - (Required) An `action_group` block as defined below.
+
+* `severity` - (Required) Specifies the severity of this Smart Detector Alert Rule. Possible values are `Sev0`, `Sev1`, `Sev2`, `Sev3` or `Sev4`.
+
+* `frequency` - (Required) Specifies the frequency of this Smart Detector Alert Rule in ISO8601 format.
+
+* `description` - (Optional) Specifies a description for the Smart Detector Alert Rule.
+
+* `enabled` - (Optional) Is the Smart Detector Alert Rule enabled? Defaults to `true`.
+
+* `throttling` - (Optional) A `throttling` block as defined below.
+
+---
+
+The `action_group` block supports the following:
+
+* `ids` - (Required) Specifies the action group ids.
+
+* `custom_email_subject` - (Optional) Specifies the custom email subject.
+
+* `custom_webhook_payload` - (Optional) A JSON String which Specifies the custom webhook payload.
```

Comment:
could we expand on what happens when this is left empty?
```suggestion
* `webhook_payload` - (Optional) A JSON String which Specifies the custom webhook payload.
```

Reply:
have added more specific explanation for this field.